src/porepy/numerics/fv/mpsa.py

```diff
@@ -384,6 +384,14 @@ def assemble_int_bound_displacement_trace(
                 * matrix_dictionary["bound_displacement_face"]
                 * proj_int.T
             )
+            cc[2, 2] -= (
```

Comment:
This imposes a symmetric stress? Short comment please.

Reply:
The comment is in the else part of the if/else. Will add a comment here also